data/reporter.js

```diff
@@ -105,16 +107,19 @@ var initialize = function(data) {
     document.initialized = true;
 };
 
-self.port.on("app_e10s_enabled", function(enabled) {
-    $("#appE10sEnabled").empty();
-    var e10s_link = document.createElement("a");
-    e10s_link.setAttribute("href", "#");
-    e10s_link.addEventListener("click", function() {
+self.port.on("app_e10s_enabled", function(appEnabled, addonEnabled) {
```

Comment:
Nit: The variable naming confused me for a second because this is not about whether the add-on is enabled or not (what the name suggests), but about e10s.